py3_analysis/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM socrata/python3
+
+RUN apt-get update
+RUN DEBIAN_FRONTEND=noninteractive apt-get install -y python-numpy python-scipy
+
+RUN pip install scikit-learn==0.17
```

Comment:
I think you'll want to include this before the pip command, to ensure we have the underlying linalg. stuff installed.

```RUN DEBIAN_FRONTEND=noninteractive apt-get -y install libblas-dev liblapack-dev gfortran```